site/src/pages/AuditPage/AuditFilter.tsx

```diff
@@ -71,9 +66,9 @@ export const AuditFilter: FC<AuditFilterProps> = ({ filter, error, menus }) => {
 					)}
 				</>
 			}
-			skeleton={
+			optionsSkeleton={
 				<>
-					<SearchFieldSkeleton />
+					<MenuSkeleton />
```

Comment:
We forgot to put a fourth `MenuSkeleton` here for the orgs dropdown

Reply:
it needs to be conditional on `showOrganizations` :^)

Reply:
Whoops...